modules/splunk-otel_kubernetes-common/common-filters.tf

```diff
@@ -0,0 +1 @@
+../../common/module/filters-smart-agent.tf
```

Comment:
someday we should rename this file.... out of scope for this PR I think, could be too complex